lib/statix/packet.ex

```diff
@@ -1,34 +1,8 @@
 defmodule Statix.Packet do
   @moduledoc false
 
-  use Bitwise
-
-  otp_release = :erlang.system_info(:otp_release)
-  @addr_family if(otp_release >= '19', do: [1], else: [])
-
-  def header({n1, n2, n3, n4}, port) do
```

Comment:
As stated on the PR description, we no longer need the header because of using `:gen_udp.send` vs `Port.command`